### ISSUE TITLE
hydra-unstable-2020-04-16 is broken on 20.03

### ISSUE THREAD

**Issue by wizeman** [NixOS/nixpkgs#89305]
**Describe the bug**

See https://hydra.nixos.org/build/120698774.

This seems to have been broken by 29cc418d945856b0ae3d25e8d7d4f3d75d176618.

Maybe #89297 needs to be backported?

**Notify maintainers**

cc @edolstra @Ma27

**Comment by Ma27** [comment]
Yeah, it's a problem that we have to use unstable Nix to keep Hydra building atm. But that's a good reason to backport #89297  IMHO.

**Comment by andir** [comment]
Running into the same issue. I added this a few days ago:

```
  nixpkgs.overlays = [
    (self: super: {
      hydra-unstable = super.hydra-unstable.overrideAttrs (_: {
        patches = [
          (super.fetchpatch {
            url = "https://github.com/NixOS/hydra/commit/15a45f1a8a867369bb58991fbcf58366b10cc62d.patch";
            sha256 = "1zmrym2wx6q4bvqxz319xnc16l484bmkqbwah5gh18q4204xdssc";
          })
        ];
      });
    })
  ];
```

**Comment by Ma27** [comment]
Just had the time to actually test a migration on my personal Hydra with a bit of live data.
It worked fine, however I found a regression in `hydra-send-stats` which I'm going to fix now before considering this mergable.

**Comment by Ma27** [comment]
Fixed by the backport of #89297 in 92d5b1b55f68971ac3c6a13af308af3a0d39e30c.
